core/src/main/java/com/linecorp/armeria/common/DefaultContextAwareEventLoop.java

```diff
@@ -130,24 +130,26 @@ public boolean isShuttingDown() {
 
     @Override
     public ScheduledFuture<?> schedule(Runnable command, long delay, TimeUnit unit) {
-        return eventLoop.schedule(context().makeContextAware(command), delay, unit);
+        return eventLoop.schedule(contextOrNull().makeContextAware(command), delay, unit);
```

Comment:
How about just calling `context()`?
Is there any reason you changed this?

Reply:
No, it was just automated refactoring.

Reply:
Then, could you revert the change in this class, please? 😄

Reply:
Yes, sorry, was having issues pulling